tools/vectorized/vectorized.c

```diff
@@ -21,16 +21,21 @@ void apply_x(Block *block, const Block *key) {
 
 // TODO: try to get a profit here
 void apply_l(Block *block) {
```

Comment:
SIMD instructions are not readable at all. Maybe here is the reason to write down pseudo-code of what we're doing as a comment before function. Smth like this:

```
/*
  v1 = *block;
  v2 = v1 + {1};
  ...
*/
void apply_l(Block *block){...}
```

Reply:
Done